app/models/katello/concerns/http_proxy_extensions.rb

```diff
@@ -5,6 +5,12 @@ module HttpProxyExtensions
 
       included do
         after_save :update_default_proxy_setting
+
+        def self.default_global_content_proxy
```

Comment:
Oh, this is a great addition!

Reply:
@jlsherrill should we refactor other queries where we try to find the global default to use this method?

Reply:
good idea! i'll see if i can find em all

Reply:
I think there will be a a couple more in https://github.com/Katello/katello/pull/8178.

Reply:
i don't actually see any in the current code base, but will change in once i rebase against #8178

Reply:
updated, let me know if i missed any